core/src/main/java/io/seata/core/rpc/netty/RpcServer.java

```diff
@@ -167,13 +172,48 @@ public Object sendSyncRequest(String resourceId, String clientId, Object message
     /**
      * Send request with response object.
      *
-     * @param channel   the channel
-     * @param message    the msg
+     * @param channel the channel
+     * @param message the msg
      * @return the object
      * @throws TimeoutException the timeout exception
      */
     @Override
     public Object sendASyncRequest(Channel channel, Object message) throws TimeoutException {
         return sendAsyncRequestWithoutResponse(channel, message);
     }
+
+    @Override
+    public void registerProcessor(int messageType, RemotingProcessor processor, ExecutorService executor) {
+        Pair<RemotingProcessor, ExecutorService> pair = new Pair<>(processor, executor);
+        this.processorTable.put(messageType, pair);
+    }
+
+    private void registerProcessor() {
+        // 1. registry on request message processor
+        ServerOnRequestProcessor onRequestProcessor =
+            new ServerOnRequestProcessor(this, getTransactionMessageHandler());
+        registerProcessor(MessageType.TYPE_BRANCH_REGISTER, onRequestProcessor, messageExecutor);
```

Comment:
You can put the registerProcessor() to parent, and use super.registerProcessor(), just look like the #2477

Reply:
Done.